Client/Frontend/Browser/HomePageHelper.swift

```diff
@@ -73,4 +73,41 @@ class HomePageHelper {
             })
         navigationController?.presentViewController(alertController, animated: true, completion: nil)
     }
+}
+
+/// Accessors for homepage details from the app state.
+/// These are pure functions, so it's quite ok to have them 
+/// as static.
+class HomePageAccessors {
+    private static let getPrefs = Accessors.getPrefs
```

Comment:
There are more than one places `getPrefs` is used in `HomePageAccessors`, so I was trying to not type `Accessors.getPrefs`.

I could've used a private 

```swift
private let getPrefs = Accessors.getPrefs
class HomePageAccessors {
```

or
```swift
class HomePageAccessors: Accessors {
```
although I prefer to not be creating a type hierarchy around simple accessor functions.